src/scenes/Scene.d.ts

```diff
@@ -6,6 +6,7 @@ import { Texture } from '../textures/Texture';
 import { WebGLRenderer } from "../renderers/WebGLRenderer";
 import { Camera } from "../cameras/Camera";
 import { RenderTarget } from "../renderers/webgl/WebGLRenderLists";
+import { WebGLRenderTarget } from "../renderers/WebGLRenderTarget";
```

Comment:
I think `WebGLCubeRenderTarget` should be more precise.

Reply:
BTW: `import { RenderTarget } from "../renderers/webgl/WebGLRenderLists";` seems incorrect to me. This should be replaced with `WebGLRenderTarget`.

Reply:
done